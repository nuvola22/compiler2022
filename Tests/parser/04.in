const a = 123;
begin end.