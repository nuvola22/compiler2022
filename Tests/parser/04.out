      program
         constant
            a
            123
         begin
